app/src/main/res/values/example_descriptions.xml

```diff
@@ -51,6 +51,7 @@
     <string name="description_styles_dds_geojson_circle_layer_clusters_description">Use GeoJSON and circle layers to visualize point data in clusters.</string>
     <string name="description_change_icon_size">Symbol layer icon size change</string>
     <string name="description_add_marker_symbol">Add simple marker symbols</string>
+    <string name="description_add_one_marker_symbol">Add a marker symbol</string>
```

Comment:
Is there a similar structure of name and description on iOS?
If yes - could we also align example name and descroption?

Reply:
Good point.